po/pl.po

```diff
@@ -0,0 +1,2133 @@
+# Polish translations for GrandOrgue package.
+# This file is put in the public domain.
+# Romuald Milewski <rommil67@gmail.com>, 2021
+msgid ""
+msgstr ""
+"Project-Id-Version: GrandOrgue\n"
+"Report-Msgid-Bugs-To: ourorgan-developers@lists.sourceforge.net\n"
+"POT-Creation-Date: 2013-02-18 09:03+0100\n"
+"PO-Revision-Date: 2013-02-16 13:30+0100\n"
+"Last-Translator: Romuald Milewski <rommil67@gmail.com>\n"
+"Language-Team: Polish <rommil67@gmail.com>\n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+msgid " (PA): "
+msgstr " (PA): "
+
+msgid " - Virtual Pipe Organ Software"
+msgstr " - Program wirtualnych organów piszczałkowych"
+
+msgid "\""
+msgstr "\""
+
+msgid "#"
+msgstr "#"
+
+#, c-format
+msgid "%.3f MB"
+msgstr "%.3f MB"
+
+#, c-format
+msgid "%.3f MB of %.3f MB"
+msgstr "%.3f MB von %.3f MB"
+
+#, c-format
+msgid "%d"
+msgstr "%d"
+
+#, c-format
+msgid "%d (%s)"
+msgstr "%d (%s)"
+
+#, c-format
+msgid "%f cent"
+msgstr "%f cent"
+
+#, c-format
+msgid "%s - left"
+msgstr "%s - lewy"
+
+#, c-format
+msgid "%s - right"
+msgstr "%s - prawy"
+
+#, c-format
+msgid "%s: %f dB"
+msgstr "%s: %f dB"
+
+#, c-format
+msgid "%s: mute"
+msgstr "%s: wycisz"
+
+msgid "&About"
+msgstr "&O"
+
+msgid "&Add"
+msgstr "&Dodaj"
```

Comment:
I'm not shure, what form of verbs is usable in the polish software: `dodaj` or `dodać`?

Reply:
My wife say both work. Safer choice might be dodaj.

Reply:
Yes, and dodaj imperative, right.

Reply:
![Screenshot (6506)](https://user-images.githubusercontent.com/79664814/135523181-100af734-6a7c-4648-9d01-6035ec99076d.png)
An example from MS Word - it is generally accepted in Poland

Reply:
Usually infinitives instead of imperatives are used in other languages in software. But if imperatives are accepted in Polland, then OK. Lets GO follow the Polish traditions.